earth_enterprise/src/server/geselectpublishroot

```diff
@@ -15,14 +15,26 @@
 # limitations under the License.
 #
 
+use warnings;
+use strict;
```

Comment:
Let's update copyright notice from `Copyright 2017 Google Inc` to `Copyright 2017 Google Inc., 2019 Open GEE Contributors`